izug/refegovservice/upgrades/20191031154412_archetypes_to_dexterity/types/EgovLeistung.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<object name="EgovLeistung"
+   meta_type="Factory-based Type Information with dynamic views"
+   i18n:domain="izug.refegovservice" xmlns:i18n="http://xml.zope.org/namespaces/i18n">
```

Comment:
✂️ this file is no longer needed. Just remove the type in `types.xml`.

Reply:
✅